google_maps/public/html/google_maps_edit.html

```diff
@@ -1,15 +1,67 @@
-<div class="wrapper-comp-settings is-active editor-with-buttons google-maps-edit-wrapper">
-  <p>Welcome to Google Maps Xblock!</p>
+{% load i18n %}
+
+<div class="wrapper-comp-settings is-active editor-with-buttons" id="google-maps-edit-wrapper">
+  <div class="xblock-inputs editor_content_wrapper">
+    <ul class="list-input settings-list">
```

Comment:
Regarding the a11y, add the following element before the `ul` element:
`<h2 class="sr">{% trans 'XBlock settings' %}</h2>`

Class `sr` will mark the element as something that will be read by a screen reader, but hidden from the users that don't use a screen reader.

This will describe the `ul` list to users using the screen reader.